integration-tests/src/lib.rs

```diff
@@ -112,11 +116,9 @@ fn para_to_system_para_sender_assertions(t: ParaToSystemParaTest) {
 	assert_expected_events!(
 		PopNetworkPara,
 		vec![
-			// Amount to reserve transfer is transferred to Parachain's Sovereign account
-			RuntimeEvent::Balances(
-				pallet_balances::Event::Withdraw { who, amount }
-			) => {
-				who: *who == t.sender.account_id,
+			// Amount to teleport is withdrawn from Sender
```

Comment:
```suggestion
			// Amount to reserve transfer is transferred to Parachain's Sovereign account
```

There is no teleport happening here as PopNet is not a trusted teleporter. I believe there was probably a change from CurrencyAdapter to FungiblesAdapter in the version uplift, which works in a different way, emitting burns/mints instead.